crawlfrontier/core/manager.py

```diff
@@ -291,7 +291,7 @@ def add_seeds(self, seeds):
                                  obj=seeds,
                                  return_classes=(list,))  # TODO: Dar vuelta
 
-    def get_next_requests(self, max_next_requests=0):
+    def get_next_requests(self, max_next_requests=0, overused_keys=OverusedKeys()):
```

Comment:
I see that the code carefully avoids mutating overused_keys variable, so using mutable default doesn't cause a bug here. But it could be better to set it to None and then set to `OverusedKeys()` in the function body, just to be on a safe side.